contract/src/contract.js

```diff
@@ -15,104 +21,184 @@ import { E } from '@agoric/eventual-send';
 const start = async (zcf) => {
   // Terms are
   const {
-    issuers: { Money: moolaIssuer },
+    issuers: { AuctionProceeds: auctionProceedsIssuer },
+    maths: { ListingFee: listingFeeMath },
```

Comment:
splitting the money issuer into separate issuers for listing fees and auction proceeds is more than we need, but I guess it doesn't hurt much.

If I were going to split the auction proceeds issuer, I'd want to do it on a per-auction basis.

Reply:
> splitting the money issuer into separate issuers for listing fees and auction proceeds is more than we need, but I guess it doesn't hurt much.

Yeah, Zoe doesn't allow the same keyword to be used under both `want` and `give`, so it's required if we want Alice to  have offer safety over giving money and receiving money from the auction.

Reply:
ah. so it's *not* more than we need.